crates/project-model/src/cfg_flag.rs

```diff
@@ -38,6 +39,18 @@ impl<'de> serde::Deserialize<'de> for CfgFlag {
     }
 }
 
+impl Serialize for CfgFlag {
```

Comment:
What are these serialization implementations used for? I couldn't find any serialization calls introduced in this PR.

Reply:
`lsp-types-0.94` requires that [`Params` implement `DeserializeOwned + Serialize`](https://github.com/gluon-lang/lsp-types/blob/master/src/request.rs#L6), which feels unfortunate in this instance.

Reply:
I've removed these implementations since I've moved to a "notify rust-analyzer that the configuration has changed"-based approach.